build/bin/release

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+if [ "$#" -lt "1" ]; then
+    echo "must specify chart directory"
+    exit 1
+fi
+
+set -x
```

Comment:
Consider using `set -eu -o pipefail`, so the script will exit on failed commands, undefined variables, or failures in a pipeline.